### PR TITLE
Add Joshua to Player Roster

### PR DESCRIPTION
**Pull request by Flaxscrip**
Hey y'all - please welcome our latest player, joining us from the EDEN pilot call.... drumroll... Joshua from FoundationX!

@foundationx - Please create a player-profile and tell us about you!

Reference / conversation: Issue #69

C.

### CODE REVIEW

File: Roles/Player/README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 * [@flaxscrip](https://flaxscrip.github.io/flaxscrip-space/)
 * [@aiggdev](https://github.com/aiggdev/aiggdev-space/)
 * [@angelabacca](https://angelabacca.github.io/angela-player-space/)
+* [@foundationx](https://github.com/foundationx)
```
Comment (macterra) [inline Roles/Player/README.md:26]: Is this a placeholder until @foundationx creates a player space?

Comment (foundationx) [reply Roles/Player/README.md:26]: Yes I will be creating a player space

Comment (Flaxscrip) [reply Roles/Player/README.md:26]: I was walking @foundationx through the Nomicon process. 

@foundationx - What David is truly saying is that I went out-of-process and should only submit the Pull Request once we've had proper discussion (in the issue). Then we make the change to the Nomicon and submit the PR for vote.

@macterra - Yes - I went ahead in my demo to show the PR step.

Comment (foundationx) [reply Roles/Player/README.md:26]: @Flaxscrip okay here is the player URL https://foundationx.github.io/foundationx-space/

**Comment by Flaxscrip** [comment]
Closed - will re-open a new PR with the proper link to Joshua's player-space.

**Pull request closed without merging**